### DISCUSSION TITLE
Add model from an engine

### DISCUSSION THREAD

**Discussion by gillien** [avo-hq/avo#1372] [Help]
We have an application with multiple engines. We have no problem generating a resource from the model of the app, but when I try with one from my engines, I get the error :

```
MyEngine::MyModelResource does not have a valid model assigned. It failed to find the MyEngine::MyModel model. 
 Please create that model or assign one using self.model_class = YOUR_MODEL
```

I tried to assign it manually and still got the same error. Any recommendation to fix that?

**Comment by adrianthedev** [comment]
Can you please post the code to the resource where you manually assign it? Also, where is the engine model located.

Thanks
